report.md

```diff
@@ -26,10 +26,74 @@ Summary in one or two sentences:
 
 There are build instructions in the projects readme. Some of the links are dead which was annoying and shows a lack of care from the projects side. However we were able to build the project without much hickup anyway.
 
+It should be noted that we have not found a way to run a specific test suite 
```

Comment:
Multiple tests can be executed with `mvn clean install -Dtest=MyTest#testMyMethod
` according to [https://strongbox.github.io/developer-guide/building-the-code.html#executing-a-particular-test](https://strongbox.github.io/developer-guide/building-the-code.html#executing-a-particular-test).

Reply:
It is possible to to run multiple tests that way but it still require us to rebuild the entire project (which is stated on the next line).

Reply:
Approving PR.